src/Documentation/grains/timers_and_reminders.md

```diff
@@ -39,19 +40,23 @@ public IDisposable RegisterTimer(
 
 # Reminders
 
-## Description
+## Reminder Description
 
 Reminders are similar to timers with a few important differences:
 
-* Reminders are persistent and will continue to trigger in all situations (including partial or full cluster restarts) unless explicitly cancelled.
+* Reminders are persistent and will continue to trigger in almost all situations (including partial or full cluster restarts) unless explicitly cancelled.
+* Reminder "definitions" are written to storage. However each specific occurrence, with its specific time, is not. This has the side effect that if the cluster is completely down at the time of a specific reminder tick, it will be missed and only the next tick of the reminder will happen.
 * Reminders are associated with a grain, not any specific activation.
 * If a grain has no activation associated with it and a reminder ticks, one will be created. e.g.: If an activation becomes idle and is deactivated, a reminder associated with the same grain will reactivate the grain when it ticks next.
 * Reminders are delivered by message and are subject to the same interleaving semantics as all other grain methods.
 * Reminders should not be used for high-frequency timers-- their period should be measured in minutes, hours, or days.
 
 ## Configuration
 
-Reminders, being persistent, rely upon storage to function. You must specify which storage backing to use before the reminder subsystem will function. The reminder functionality is controlled by the SystemStore element in the server-side configuration. It works with either Azure Table or SQL Server as the store.
+Reminders, being persistent, rely upon storage to function.
+You must specify which storage backing to use before the reminder subsystem will function.
+The reminder functionality is controlled by the SystemStore element in the server-side configuration.
```

Comment:
This is outdated - SystemStore is from XML config that we deprecated. We should replace this and the next line with something like: "This is done by configuring one of the reminder providers via `UseXReminderService` extension methods."

Reply:
This is the new content of the Configuration section:

## Configuration

Reminders, being persistent, rely upon storage to function.
You must specify which storage backing to use before the reminder subsystem will function.
This is done by configuring one of the reminder providers via `UseXReminderService` extension methods.

Azure Table configuration:
...